Add CLI pages for hpopt and fingerprint (#914)
chemprop/chemprop: docs/source/tutorial/cli/fingerprint.rst

.. _fingerprint:

Fingerprint
============================

To calculate the learned representations (encodings) of model inputs from a pretrained model, run

.. code-block::
   
   chemprop fingerprint --test-path <test_path> --model-path <model_path> 

> where :code:`<test_path>` is the path to the CSV file containing SMILES strings, and :code:`<model_path>` is the location of checkpoint(s) or model file(s) to use for prediction. It can be a path to either a single pretrained model checkpoint (.ckpt) or single pretrained model file (.pt), a directory that contains these files, or a list of path(s) and directory(s). If a directory, will recursively search and predict on all found (.pt) models. By default, predictions will be saved to the same directory as the test path. If desired, a different directory can be specified by using :code:`--output <path>`

Reviewer: ```suggestion
where :code:`<test_path>` is the path to the CSV file containing SMILES strings, and :code:`<model_path>` is the location of checkpoint(s) or model file(s) to use for prediction. It can be a path to either a single pretrained model checkpoint (.ckpt) or single pretrained model file (.pt), a directory that contains these files, or a list of path(s) and directory(s). If a directory, will recursively search and predict on all found (.pt) models. By default, predictions will be saved to the same directory as the test path. If desired, a different directory can be specified by using :code:`--output <path>`. The output <path> can end with either .csv or .npz, and the output will be saved to the corresponding file type.
```

Author: thanks. Is this change true also for `predict` (`preds-path`)?

.. _fingerprint:

Fingerprint
============================

To calculate the learned representations (encodings) of model inputs from a pretrained model, run

.. code-block::
   
   chemprop fingerprint --test-path <test_path> --model-path <model_path> 

> where :code:`<test_path>` is the path to the CSV file containing SMILES strings, and :code:`<model_path>` is the path to the trained model (.ckpt or .pt file) or a directory containing the trained model files. By default, predictions will be saved to the same directory as the test path. If desired, a different directory can be specified by using :code:`--output <path>`

Reviewer 2: ```suggestion
where :code:`<test_path>` is the path to the CSV file containing SMILES strings, and :code:`<model_path>` is the location of checkpoint(s) or model file(s) to use for prediction. It can be a path to either a single pretrained model checkpoint (.ckpt) or single pretrained model file (.pt), a directory that contains these files, or a list of path(s) and directory(s). By default, predictions will be saved to the same directory as the test path. If desired, a different directory can be specified by using :code:`--output <path>`
```

Reviewer: We should also mention that when a directory is provided, it only includes the model file (.pt)

Reviewer 2: @hwpang I agree and I just edited the suggested code.

For example:

.. code-block::
  

>     chemprop predict --test-path tests/data/smis.csv \
>         --model-path tests/data/example_model_v2_regression_mol.ckpt \
>         --output fps.csv

Reviewer 2: ```suggestion
    chemprop fingerprint --test-path tests/data/smis.csv \
    --model-path tests/data/example_model_v2_regression_mol.ckpt \
    --output fps.csv
```

Reviewer 2: I am trying to make this consistent with what we used in the training documentation. If you think it makes sense, please also change the one in the prediction documentation.
